auth0_flutter/example/analysis_options.yaml

```diff
@@ -73,7 +73,7 @@ linter:
     join_return_with_assignment: true
     leading_newlines_in_multiline_strings: true
     library_private_types_in_public_api: true
-    lines_longer_than_80_chars -> packages only: true
+    lines_longer_than_80_chars: true
```

Comment:
Could you please change `todo` to `warning` on L29? Also for the  other YAML files.

Reply:
Done in 110bd0c